examples/with-hasura-thirdpartyemailpassword/README.md

```diff
@@ -29,7 +29,7 @@ npm install
 ngrok http 3001
```

Comment:
Shouldn't the example apps be updated to use react 18? You can change the supertokens-auth-react dependency in all the example apps to point to the branch which has the react 18 compatibility. Then once we release it, we can change it to use npm.

Reply:
They have been updated in #439